typings/rendition.d.ts

```diff
@@ -1,242 +1,12 @@
 declare module 'rendition' {
```

Comment:
Can we completely delete this file?

Reply:
We should, left it for now in order to tackle the 'Form' typings a bit later.